h/services/group_members.py

```diff
@@ -53,7 +53,7 @@ def get_memberships(
             select(GroupMembership)
             .join(User)
             .where(GroupMembership.group == group)
-            .order_by(User.username)
+            .order_by(nulls_first(GroupMembership.created), User.username)
```

Comment:
I was going to comment that .desc() and nulls_last made more sense (to me) but I've seen this is the order described on the issue.